test/reconciler-tests.sh

```diff
@@ -27,7 +27,7 @@ if [ "${EVENTING_KAFKA_BROKER_CHANNEL_AUTH_SCENARIO:-""}" != "" ]; then
   success
 fi
 
-go_test_e2e -timeout=1h ./test/e2e_new/... || fail_test "E2E (new) suite failed"
+go_test_e2e -timeout=1h -run TestDeadLetterSinkExtensions ./test/e2e_new/... || fail_test "E2E (new) suite failed"
```

Comment:
can we revert this change?

Reply:
I suggested this to get feedback faster from CI, we will revert it once it passes